modules/javafx.base/src/main/java/javafx/collections/ListChangeListener.java

```diff
@@ -231,11 +231,9 @@ public boolean wasUpdated() {
          * Returns a subList view of the list that contains only the elements added. This is actually a shortcut to
          * <code>c.getList().subList(c.getFrom(), c.getTo());</code>
          *
-         * <pre>{@code
-         * for (Node n : change.getAddedSubList()) {
+         * <pre><code> for (Node n : change.getAddedSubList()) {
```

Comment:
What is the problem with the existing example? It already matches our preferred pattern for example code:

```
* <pre>{@code
* sample code
* goes here
* }</pre>
```

Reply:
Oh, in Eclipse that was not rendered correctly, probably because of the interaction between `<pre>` and `{@code}`. Do you want me to revert?

Reply:
Yes, please.